### ISSUE TITLE
Angular ESLint Utils

### ISSUE THREAD

**Issue by timdeschryver** [timdeschryver/eslint-plugin-ngrx#196]
This issue is here to remind us to take a look at the Angular ESLint Utils (https://github.com/angular-eslint/angular-eslint/pull/676) and maybe refactor some of our code to use those utils where possible.

**Comment by timdeschryver** [comment]
@rafaelss95 I noticed you've creating an issue at eslint-rxjs to allow suggestions in  `fromFixture`.
My idea was to use the test utils from the angular-eslint, do you have an opinion about this?

**Comment by rafaelss95** [comment]
Yes, I opened a problem there because it would help us at the moment, since we are currently using `fromFixture`, but I have no objections to using [**`convertAnnotatedSourceToFailureCase`**](https://github.com/angular-eslint/angular-eslint/blob/9dc44c14b101dd4863447c2466fad3f1c2c5e03c/packages/utils/src/test-helpers.ts#L142).
